examples/github-complete/variables.tf

```
@@ -14,6 +14,18 @@ variable "alb_ingress_cidr_blocks" {
   type        = list(string)
 }
 
+variable "alb_enable_deletion_protection" {
+  description = "If true, deletion of the load balancer will be disabled via the AWS API. This will prevent Terraform from deleting the load balancer. Defaults to false."
+  type        = bool
+  default     = false
```

lets set these to `null` for defaults to rely on whatever is setup upstream or by the API
```suggestion
  default     = null
```